### PR TITLE
UI enhancements: Nav active state, Copy button UI and Text colors

### PR DESCRIPTION
**Pull request by rushenn**
Changes:
- Add platform based active state on content navigation links.
- Update copy button UI.
- Update footer color on dark mode.
- Add hover color option to the nav links on dark mode.

### CODE REVIEW

File: source/_templates/content-navigation.html [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
     <div class="container">
         <nav class="content__nav__inner">
-            <a href="" class="active" data-content-nav="cn-kubernetes">
+            <a href="" class="{{ 'active' if doc_platform == 'kubernetes' }}" data-content-nav="cn-kubernetes">
                 <img class="hidden-rm" src="{{ pathto('_static/img/icons/kubernetes.svg',1) }}" alt="Kubernetes" />
                 <img class="hidden-rm" src="{{ pathto('_static/img/icons/kubernetes-inactive.svg',1) }}" alt="Kubernetes" />
                 Kubernetes
             </a>
 
-            <a href="" data-content-nav="cn-docker">
```
Comment (ravindk89) [inline source/_templates/content-navigation.html:10]: `make container` <- keeping it generic to account for Podman as well as Docker.

**Review by ravindk89 — APPROVED** [review]
LGTM mod one nit to ensure the K8s platform dropdown only shows up when on the K8s page.

As a note, for Phase 1 we will only have Upstream so we can probably hide that sub-banner before Go-Live and then uncomment each platform as we add in support.

**Comment by rushenn** [comment]
@ravindk89 Thank you for the review. I have addressed the comments, PTAL.

**Comment by ravindk89** [comment]
LGTM - I'll squash and merge

**Pull request merged**